src/pages/settings/AddRole/AddRoleView.tsx

```diff
@@ -19,7 +19,7 @@ function AddRoleView() {
 				<Grid item xs={12}>
 					<Paper>
 						<Box p={2}>
-							<AddRoleForm />
+				
```

Comment:
If you have removed `AddRoleForm` component, then what is the point of keep `Box` component?